docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -661,6 +661,169 @@ public class PersonRepository implements PanacheRepositoryBase<Person,Integer> {
 }
 ----
 
+== Mocking
+
+=== Using the “entity pattern”
```

Comment:
The documentation now refers to it as the "active record pattern". And I didn't put quote in it so if you want your quote please update the other titles ;)

Reply:
Grr, I much prefer "entity pattern" over "active record pattern" because those are not active records.